Sources/SecureEnclave.swift

```diff
@@ -163,9 +165,11 @@ public final class SecureEnclave {
             return .success(string)
             
         case let .error(status):
-            let userCancelled = (status == errSecUserCanceled || status == errSecAuthFailed)
-            if userCancelled {
-                return .userCancelled
+            let authFailed = (status == errSecAuthFailed)
+            let userCancelled = (status == errSecUserCanceled)
+            if userCancelled || authFailed {
```

Comment:
Is there a reasonable way to consolidate this logic with the duplicate logic on 110? Feels like this might be a bit fragile over time.